### ISSUE TITLE
missing  openvasmd.sock 

### ISSUE THREAD

**Issue by neginsadeghi** [immauss/openvas#68]
I can't find openvasmd.sock file on the container, can you please guide me that I could find it ?
or force gvm to create a unix socket ? 

I want to use API and that's why I need it

**Comment by immauss** [comment]
Make sure you are looking at current docs on using the API. I'm pretty sure that socket name is no longer used. 
There is /tmp/ospd.sock if you want to work dicrectly with the openvas scanner. For the gvmd, you don't need the socket to access the API.

**Comment by maxproske** [comment]
@neginsadeghi The unix socket is located at `/var/run/openvasmd.sock`

**Comment by immauss** [comment]
@maxproske  Yes. ... Thank you. I've been underwater and getting pulled deeper by the blasted bug that was causing my container to segfault.

**Comment by dotw** [comment]
@immauss Hi, did u get any progress? I have the same issue. the api services report Premature end of file and I have no idea to fix it.

**Comment by immauss** [comment]
@dotw Not sure what you are referencing. Can you open a new issue and provide some logs? 
And know that I pushed a new version yesterday, 21.4.4-06 that fixes quite a lot of problems, so if you could try to reproduce the problem with the new image  first, I would really appreciate it.

**Comment by maxproske** [comment]
@dotw If by "the api services" you mean Mixeway's MixewayOpenVASRestAPI, they don't appear to support gvmd listening on a port instead of a socket. You can follow my issue here: https://github.com/Mixeway/MixewayOpenVASRestAPI/issues/8
